src/_repobee/command/repos.py

```diff
@@ -176,7 +170,7 @@ def _create_state_separated_push_tuples(
     teams: List[plug.Team],
     template_repos: List[plug.TemplateRepo],
     api: plug.PlatformAPI,
-) -> Tuple[List[Push], List[Push]]:
+) -> Tuple[List[git.Push], List[git.Push]]:
```

Comment:
I want to avoid using qualified names in type hints, it makes it harder to read.

```suggestion
) -> Tuple[List[Push], List[Push]]:
```